airbyte-scheduler/scheduler-persistence/src/main/java/io/airbyte/scheduler/persistence/DefaultJobPersistence.java

```diff
@@ -802,7 +802,7 @@ private static void truncateTable(final DSLContext ctx, final String schema, fin
     final Table<Record> backupTableSql = getTable(backupSchema, tableName);
     ctx.dropTableIfExists(backupTableSql).execute();
     ctx.createTable(backupTableSql).as(DSL.select(DSL.asterisk()).from(tableSql)).withData().execute();
-    ctx.truncateTable(tableSql).restartIdentity().execute();
+    ctx.truncateTable(tableSql).restartIdentity().cascade().execute();
```

Comment:
i'm a little concerned about this change, since this function seems to be truncating and backing up tables, but the associated rows that are deleted from the sync_stats table will not be backed up. It looks like this is called when exporting a db to a new location (possibly?) and this will work as long as the tables are moved in the correct order. Not sure if this is ok to live with for now or not.

Reply:
cc: @ChristopheDuong since it looks like you wrote most of this

Reply:
The "backup" logic was introduced at a time when almost everything was persisted in a single table (json blobs?). But now that we are splitting more things out, cascading may make sense but the "backup" logic is probably deprecated...

IIRC backing things up was mainly due to being able to roll back if migrations were not successful, but again since this was written, the migration module/code also evolved.